test/core.test.ts

```diff
@@ -158,761 +159,790 @@ const customPath = "m/0'/0/0/0";
 const customPrivateKey = new Uint8Array([
-    54,
-    89,
-    88,
-    225,
-    226,
-    154,
-    80,
-    20,
-    212,
-    54,
-    118,
-    144,
-    130,
-    94,
-    70,
-    246,
-    252,
-    199,
-    82,
-    236,
-    214,
-    238,
-    126,
-    196,
-    85,
-    224,
-    18,
-    25,
-    144,
-    98,
-    202,
-    220
+  54,
+  89,
+  88,
+  225,
+  226,
+  154,
+  80,
+  20,
+  212,
+  54,
+  118,
+  144,
+  130,
+  94,
+  70,
+  246,
+  252,
+  199,
+  82,
+  236,
+  214,
+  238,
+  126,
+  196,
+  85,
+  224,
+  18,
+  25,
+  144,
+  98,
+  202,
+  220,
 ]);
 
 const customPublicKey = new Uint8Array([
-    3,
-    147,
-    159,
-    182,
-    106,
-    235,
-    134,
-    53,
-    180,
-    193,
-    187,
-    140,
-    166,
-    15,
-    246,
-    28,
-    238,
-    201,
-    108,
-    77,
-    92,
-    123,
-    85,
-    2,
-    209,
-    107,
-    254,
-    149,
-    22,
-    211,
-    68,
-    37,
-    74
+  3,
+  147,
+  159,
+  182,
+  106,
+  235,
+  134,
+  53,
+  180,
+  193,
+  187,
+  140,
+  166,
+  15,
+  246,
+  28,
+  238,
+  201,
+  108,
+  77,
+  92,
+  123,
+  85,
+  2,
+  209,
+  107,
+  254,
+  149,
+  22,
+  211,
+  68,
+  37,
+  74,
 ]);
 
 const tx = {
-    'msg':  [
-        {
-            'type':  'cosmos-sdk/MsgSend',
-            'value': {
-                'from_address': cosmosAddress,
-                'to_address':   cosmosAddress,
-                'amount':       [
-                    {
-                        'denom':  'stake',
-                        'amount': '1000000'
-                    }
-                ]
-            }
-        }
-    ],
-    'fee':  {
-        'amount': [
+  type: 'cosmos-sdk/StdTx',
+  value: {
+    msg: [
+      {
+        type: 'cosmos-sdk/MsgSend',
+        value: {
+          from_address: cosmosAddress,
+          to_address: cosmosAddress,
+          amount: [
             {
-                'denom':  'stake',
-                'amount': '1'
-            }
-        ],
-        'gas':    '100'
+              denom: 'stake',
+              amount: '1000000',
+            },
+          ],
+        },
+      },
+    ],
+    fee: {
+      amount: [
+        {
+          denom: 'stake',
+          amount: '1',
+        },
+      ],
+      gas: '100',
     },
-    'memo': 'This is a test.'
+    memo: 'This is a test.',
+  },
 };
 
 const signMeta = {
-    'account_number': '1',
-    'chain_id':       'cosmos',
-    'sequence':       '0'
+  account_number: '1',
+  chain_id: 'cosmos',
+  sequence: '0',
 };
 
 const knownSignMsg = {
-    'account_number': signMeta['account_number'],
-    'chain_id':       signMeta['chain_id'],
-    'fee':            tx['fee'],
-    'memo':           tx['memo'],
-    'msgs':           tx['msg'],
-    'sequence':       signMeta['sequence']
+  account_number: signMeta['account_number'],
+  chain_id: signMeta['chain_id'],
+  fee: tx.value.fee,
+  memo: tx.value.memo,
+  msgs: tx.value.msg,
+  sequence: signMeta['sequence'],
 };
```

Comment:
I see that the sample tx objects have been restructured, but it's a bit difficult to dig into changes in the tests since there's a lot of automatic formatting updates. Which tests have had relevant updates?

Reply:
https://github.com/Kava-Labs/sig/pull/1/files?w=1 helps a bit. However, the only changes to tests we're to restructure the transaction objects to change `tx.[field]` to `tx.value.[field]`